sdk/dnsresolver/Microsoft.Azure.Management.DnsResolver/tests/TestDataGenerator.cs

```diff
@@ -0,0 +1,184 @@
+// ------------------------------------------------------------------------------------------------
+// <copyright file="TestDataGenerator.cs" company="Microsoft Corporation">
+//   Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------------------------
+
+namespace DnsResolver.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Management.DnsResolver;
+    using Microsoft.Azure.Management.DnsResolver.Models;
+    using Microsoft.Azure.Management.Resources.Models;
+    using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+
+    internal static class TestDataGenerator
+    {
+        private const string DefaultResourceLocation = "Central US";
+
+        private static readonly Random Random;
+
+        static TestDataGenerator()
+        {
+            Random = new Random();
```

Comment:
Where is this used?  You will want to make sure that either you use a random seed here (which cna be stored and reused from the test recording), or that any values you generate with this during live testing are retrieved from the test recording during playback

Reply:
Yes, we are aware the randomly generated data may lead to failure to test failures. 
The way we uses TestDataGenerator is to generate object name and objects that can be recorded and playback,  
Also did a sanity check on this issue.